docs/it/Il-bias-del-fare-codice-a-vita.md

```diff
@@ -0,0 +1,19 @@
+# Il bias del fare codice a vita
+
+La carriera di uno sviluppatore è complessa da molti punti di vista e non priva di incognite e, al primo posto, possiamo mettere il fatto che sia una categoria nuova e in continua evoluzione. Basti guardare ad alcuni schemi che rappresentano la crescita e le conoscenze di una certa mansione che ci troviamo davanti ad un albero di scelte impressionante. Cercando di allontanarsi un poco per vedere un percorso di massima non possiamo non pensare alla crescita verticale verso mansioni mano a mano più gestionali e meno operative, è un pensiero che prima o poi si tende a fare. Questo è legato non solo all’ambizione ma anche ad un pensiero sul proprio futuro, in generale sull’invecchiamento intellettuale e sulle conseguenze che inevitabilmente può avere sul nostro rendimento, sulla nostra freschezza e sulla capacità di stare dietro ad un mondo, quello informatico, in perenne vorticoso movimento; questo rendimento è legato a doppio filo con il nostro intelletto.
+
+Dal punto di vista della catalogazione l’intelligenza, come capacità di risolvere un problema, viene descritta come Fluida o Cristallizzata (teorizzata dallo psicologo Cattell). La prima, quella fluida, è il pensiero logico, la nostra capacità di risolvere problemi nuovi. Quella cristallizzata, invece, è la nostra capacità di utilizzare le conoscenze pregresse; questa è quella che di solito cade sotto il nome di “seniority”. Nel lavoro dello sviluppatore sono importanti entrambe probabilmente in egual misura.
+
+Il passare del tempo è intuitivo che vada a beneficio dell'intelligenza cristallizzata perché ci da un bacino più ampio a cui attingere. In passato a questa catalogazione veniva legata al Q.I. da cui prende anche il nome: era il rapporto tra età psicologica e anagrafica (da cui quoziente). L’intelligenza fluida, invece, ha una caratteristica inversa, cioè con l’età ha una tendenza a calare, man mano che l’età avanza caliamo in freschezza. Le nostre performance come velocità, capacità di visualizzazione ed elaborazione sono legate a questi due elementi che sommati formano il nostro intelletto.
```

Comment:
```suggestion
Il passare del tempo va intuitivamente beneficio dell'intelligenza cristallizzata perché ci da un bacino più ampio a cui attingere. In passato questa catalogazione veniva legata al Q.I. da cui prende anche il nome: era il rapporto tra età psicologica e anagrafica (da cui quoziente). L’intelligenza fluida, invece, ha una caratteristica inversa, cioè con l’età ha una tendenza a calare, facendoci perdere freschezza. Le nostre performance come velocità, capacità di visualizzazione ed elaborazione sono legate a questi due elementi che sommati formano il nostro intelletto.
```

Reply:
@Cadienvan ho provato a effettuare il commit del suggerimento sia in questa che sulle altre ma ottengo sempre un "Could not apply suggestion." e mi invita a fare refresh, hai mica idea del perchè?